### PR TITLE
1630 notes health care eval asset total 

### PR DESCRIPTION
**Pull request by MeganGeissler**
-Casenotes total assets by member 
-Dialog displays total assets by member
-For cars, exclude Use Code 4 and 5 from asset total. For all other codes, exclude the highest valued vehicle.
-For real estate, do not sum those with Property Status of "home residence"
-Fixed spelling issue 
-Fixed secu_string casenoting

### CODE REVIEW

**Review by C-Love — CHANGES_REQUESTED** [review]
The information about asset amounts and totals should probably be in the dialog as well, so the worker has a way to view the information before confirming.

File: notes/health-care-evaluation.vbs [hunk 1]
```diff
@@ -3564,9 +3562,102 @@ function write_asset_details_in_NOTE()
...
+
+		total_sum = car_1_total + car_2_total + car_3_total
+		
+		'Removing highest value vehicle if more than 1 vehicle 
+		car_value = Array(car_1_total, car_2_total, car_3_total)
+		If car_count > 1 Then 
+			For value = 0 to Ubound(car_value)
+				If car_value(value) >maxVal THen 
+					maxVal = car_value(value)
+					msgbox maxVal & "HI"
```
Comment (MDR22) [inline notes/health-care-evaluation.vbs:3627]: Can remove testing msgbox

**Review by MDR22 — APPROVED** [review]
I agree with Casey's comment. Had one comment about a msgbox, otherwise looks good to me!

**Comment by MeganGeissler** [comment]
@MDR22 & @C-Love - thanks, i've made the adjustments suggested! Feel free to test again if you have time.

**Review by C-Love — APPROVED** [review]
Looks great to me!

**Pull request merged**